test/fixtures/payload-helpers/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG BUILDKITE_BRANCH
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:${BUILDKITE_BRANCH}-ci
```

Comment:
Should this (and a few other places) now be `$BRANCH_NAME` to fix the issue where branches with forward slashes don't pass?

Reply:
Yes, I'll have to merge up.